api/blueprints/routes.py

```diff
@@ -0,0 +1,74 @@
+from flask import Blueprint, request, jsonify
```

Comment:
This looks like a great start 👍 Let's refine this once we have a frontend prototype ready that can interact with the data.